README.md

```diff
@@ -26,12 +25,12 @@ Then run docker compose without the WebAPI project.
 docker compose -f docker-compose-no-webapi.yml up 
 ```
 
-
 ## DB development
 This project uses Entity Framework core to manage DB migrations. DB development can ether be done through Visual Studios Package Manager Console (PMC), or through the CLI. 
 
 ### DB development through PMC
-Set Digdir.Domain.Dialogporten.Infrastructure as startup project in Visual Studios solution explorer, and as default project in PMC. You are now ready to use [EF core tools through PMC](https://learn.microsoft.com/en-us/ef/core/cli/powershell). Run the following command for more information:
+Set Digdir.Domain.Dialogporten.Infrastructure as startup project in Visual Studios solution explorer, and as default project in PMC. You are now ready to use [EF core tools through PMC](https://learn.microsoft.com/en-us/ef/core/cli/powershell).<br>
```

Comment:
Kan bruke dobbel space på slutten av linjer i stedet for <br> for linebreaks i markdown ^_^